paper.md

```diff
@@ -84,6 +84,8 @@ Other general purpose packages, written in MATLAB, provide similar functionality
 
 # Acknowledgements
 
-We acknowledge Liberty Hamilton for her work [@HamiltonEtAl2017] which was especially helpful to use as a comparison and to build off conceptually in the development of this project. This work is supported by NIH-R01NS104585 (EL,AG; PI: M. Hamalainen).
+We acknowledge Liberty Hamilton for her work [@HamiltonEtAl2017] which was especially helpful to use as a comparison and to build off conceptually in the development of this project.
+
+This work is supported by the Renée James Seed grant to Accelerate Scientific Research from University of Oregon (APR, NCS; PI: N. Swann), NIH-1RF1MH117155-01 (AMR, PI: A. Raslan), NIH-R01NS104585 (EL, AG; PI: M. Hämäläinen) and [CZI-EOSS2](https://chanzuckerberg.com/eoss/proposals/improving-usability-of-core-neuroscience-analysis-tools-with-mne-python/) (EL, AG; PI: D. McCloy).
```

Comment:
I thought Dan was on this paper. EOSS2 just funds him. So since he's not an author, we can just omit it. Sorry for the noise!
```suggestion
This work is supported by the Renée James Seed grant to Accelerate Scientific Research from University of Oregon (APR, NCS; PI: N. Swann), NIH-1RF1MH117155-01 (AMR, PI: A. Raslan), NIH-R01NS104585 (EL, AG; PI: M. Hämäläinen).
```